hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -69,6 +69,18 @@
       try {
         this.output = fs.append(path, bufferSize);
       } catch (RemoteException e) {
+        if (e.getMessage().contains("not sufficiently replicated yet")) {
```

Comment:
Question : I see that RemoteException supports optional error-codes. Not sure if you see an error-code for this case. If you do see an error-code, can we use that to match instead of string matching ?

Reply:
If you take a look at the error-codes, they are generic. To give you an example I ran the case to show the error code generated :

<img width="1013" alt="screen shot 2018-05-02 at 2 59 59 pm" src="https://user-images.githubusercontent.com/2722167/39551476-fe0b1a08-4e19-11e8-98e5-f72c1d7531e3.png">

It's best to not rely on the generic error code.

Reply:
Sounds good. Thanks for the snapshot.

Reply:
can we pull the err msg string into a final variable..